TypeScript.py

```diff
@@ -792,6 +795,29 @@ def on_close(self, view):
                     cli.service.close(view.file_name())
         logger.view_debug(view, "exit on_close")
 
+    def handlePulse(self):
+        view = active_view()
+        info = self.getInfo(view)
+        if info:
+            self.showTopLineStatus(info, view)
+        sublime.set_timeout(self.handlePulse, 500)
+
+    def showTopLineStatus(self, info, view):
+        if not self.pulse_started:
+            self.pulse_started = True
+            sublime.set_timeout(self.handlePulse, 500)
+        visStart = view.line(view.visible_region().begin())
+        if (info.prevVisStart != visStart) or (info.prevHasErrors != info.hasErrors):
+            info.prevVisStart = visStart
+            info.prevHasErrors = info.hasErrors
+            scope = "entity.name.function"
+            if info.hasErrors:
+                scope = "keyword"
```

Comment:
Yes.  Do you know how to do that?  I am wondering if we can augment whatever is the current theme with an additional theme so that we can define colors.  The colors aren't actually used in this case because the icon supplies its own colors.